### PR TITLE
lab/niriss fixes

### PR DESCRIPTION
**Pull request by lboogaard**
Dear Gabe,
Here is a long overdue PR with some fixes needed to run NIRISS data
- don't open local CRDS files in update mode (fails if they are read only, as on our servers)
- skip failing header verification (workaround for #117)
- fix `n-clear` to `clear` because it won't work otherwise

FYI: @jdavies-st @rameyer @TheSkyentist

### CODE REVIEW

**Review by TheSkyentist — COMMENTED** [review]
For some reason this also works with `key = filt.lower()` and I can't understand why this works as well.

File: grizli/pipeline/auto_script.py [hunk 1]
```diff
@@ -4572,7 +4572,8 @@ def make_filter_combinations(root, weight_fnu=2, filter_combinations=FILTER_COMB
             head[band] = im_i[0].header.copy()
         else:
             for k in im_i[0].header:
-                head[band][k] = im_i[0].header[k]
+                #head[band][k] = im_i[0].header[k]
```
Comment (TheSkyentist) [inline grizli/pipeline/auto_script.py:4575]: You can also just skip the offending keys:
`if k == '' or k == 'HISTORY': continue`

Comment (lboogaard) [reply grizli/pipeline/auto_script.py]: sounds good, I can change that

**Comment by lboogaard** [comment]
Note: https://github.com/gbrammer/grizli/blob/3b3f09da271800163443c0d09091a35a1aa6d81f/grizli/aws/visit_processor.py#L2192

**Comment by gbrammer** [comment]
Adding that `n` to the NIRISS filters is a bit of an ugly hack, but it's assumed to be set as such in various places.  Does this change affect that or should it be back compatible?

**Comment by gbrammer** [comment]
And for the background image, maybe you can rather write the updated version to the working directory and update the returned `bkg_file` path accordingly.

**Comment by lboogaard** [comment]
Thanks - Raphael and I are looking into this

**Comment by gbrammer** [comment]
Is this PR now superseded by the others @TheSkyentist submitted today?

**Comment by TheSkyentist** [comment]
I believe so but I'll let @lboogaard be the final decider. The "n-clear" issue still exists (see #191) though I don't think this PR presents a global solution for combined NIRCam/NIRISS data.

**Comment by lboogaard** [comment]
I agree - let's close this in favor of #191

**Pull request closed without merging**